ttnn/cpp/ttnn/operations/functions.hpp

```diff
@@ -554,12 +552,12 @@ static bool allclose(const Tensor& tensor_a, const Tensor& tensor_b, Args... arg
     auto tensor_b_buffer = tt::tt_metal::owned_buffer::get_as<DataType>(tensor_b);
 
     for (int index = 0; index < tensor_a_buffer.size(); index++) {
-        if (not detail::nearly_equal(tensor_a_buffer[index], tensor_b_buffer[index], args...)) {
+        using ::ttnn::detail::nearly_equal;
```

Comment:
FYI - without fully qualifying I was running into https://pastecode.io/s/3iiga3to

Fully qualifying inline, clang-format used the ugly space-less format:
```
if (not::ttnn::detail::nearly_equal(...)) {
```